sphinx/ext/apidoc.py

```diff
@@ -0,0 +1,439 @@
+# -*- coding: utf-8 -*-
```

Comment:
It seems this is a new file.
Please rename from `sphinx/apidoc.py` and create `sphinx/apidoc.py` as a new file to keep commit history,